cypress/.eslintrc.js

```diff
@@ -3,15 +3,14 @@
 module.exports = {
   parser: 'babel-eslint',
   extends: [
-    'plugin:cypress/recommended',
     'eslint:recommended',
     'plugin:import/errors',
     'plugin:import/warnings',
-    'preact',
+    'plugin:cypress/recommended',
```

Comment:
I put Cypress just before prettier because the order of them determines which eslint rule wins if two configurations have the same rule. I also removed Preact from here, because it is not relevant to tests.